qgis-app/base/validator.py

```diff
@@ -4,37 +4,34 @@
 from django.core.exceptions import ValidationError
 from django.utils.translation import ugettext_lazy as _
 
-GPKG_MAX_SIZE = getattr(settings, 'GPKG_MAX_SIZE', 1000000)  # 1MB
+RESOURCE_MAX_SIZE = getattr(settings, 'RESOURCE_MAX_SIZE', 1000000)  # 1MB
 
 
-def gpkg_validator(gpkg_file) -> bool:
-    """GeoPackage File Validation"""
-
+def filesize_validator(file) -> bool:
     try:
-        if gpkg_file.getbuffer().nbytes > GPKG_MAX_SIZE:
+        if file.getbuffer().nbytes > RESOURCE_MAX_SIZE:
             raise ValidationError(
```

Comment:
@sumandari I think it would be better if you write a custom validation error that returns this error message.